lib/cdc/util/relational_event.go

```diff
@@ -111,13 +113,16 @@ func (s *SchemaEventPayload) GetData(pkMap map[string]any, tc *kafkalib.TopicCon
 		retMap[constants.DatabaseUpdatedColumnMarker] = s.GetExecutionTime().Format(ext.ISO8601)
 	}
 
-	// Iterate over the schema and identify if there are any fields that require extra care.
-	afterSchemaObject := s.Schema.GetSchemaFromLabel(cdc.After)
-	if afterSchemaObject != nil {
-		for _, field := range afterSchemaObject.Fields {
+	return retMap
+}
+
+// parseValues will take `retMap` and `kind` (which part of the schema should we be inspecting) and then parse the values accordingly.
+// This will unpack any Debezium-specific values and convert them back into their original types.
+func (s *SchemaEventPayload) parseValues(retMap map[string]any, kind cdc.FieldLabelKind) map[string]any {
```

Comment:
Do we need to return anything since this is going to just mutate `retMap`?

Reply:
I feel like having this method return a value makes it seem like the `retMap` that is passed in isn't going to be changed.

Reply:
I returned it because having the `retMap` be mutated in place seems brittle

Reply:
And I didn't think we should create a new one since it'll increase the space complexity

Reply:
Isn't it going to be mutated in place though since `map[string]any` is going to be passed by reference and not value?

Reply:
Def agree that having it be mutated in place seems brittle.

Reply:
It is, but we're not relying on it being updated in-place since it's being returned as well.

Reply:
That seems a little dangerous to me though because it has an unexpected side effect that is not immediately obvious: that `s.Payload.Before`/ `s.Payload.After` will be modified after calling this method. I'd rather have it be obvious that the mutation is going to happen in place.